03-client-api/01-java.md

```diff
@@ -8,7 +8,22 @@ templatePath: 03-client-api/references/
 
 ## Installation
 
-#### To use this client, you need a compatible TypeDB Server running. Visit our [Compatibility Table](#version-compatibility)
+### Prerequisites
+
+To use this client, you need a compatible version of TypeDB Server running. Please see the
+[Compatibility Table](#version-compatibility) to check what version do you need, depending on the TypeDB server version
+being used.
+
+Maven is expected to be used as a build system for the Java project.
```

Comment:
I'm thinking we should just delete this sentence. It sounds like we're "requiring" or at least "expecting" our users to use Maven. The reality is that many use Gradle, and they are most likely accustomed to converting a `pom.xml` specification to a Gradle dependency spec (which is much terser!)